ee/docker-ee/centos.md

```diff
@@ -64,28 +62,24 @@ apply:
   solid-state media (SSD) is recommended. Do not start Docker until properly
   configured per the [storage guide](/storage/storagedriver/device-mapper-driver/){: target="_blank" class="_" }.
 
-### Find your Docker Engine - Enterprise repo URL
+### Locating the URL for the Docker Engine - Enterprise Repo
 
-To install Docker Enterprise, you will need the URL of the Docker Enterprise repository associated with your trial or subscription:
+The product URL associated with your trial or subscription is required to install Docker Engine - Enterprise, which can be attained using the following procedure (note that these instructions apply to Docker on CentOS and for Docker on Linux, which includes access to Docker Engine - Enterprise for all Linux distributions):
 
-1.  Go to [https://hub.docker.com/my-content](https://hub.docker.com/my-content){: target="_blank" class="_" }. All of your subscriptions and trials are listed.
-2.  Click the **Setup** button for **Docker Enterprise Edition for {{ linux-dist-long }}**.
-3.  Copy the URL from **Copy and paste this URL to download your Edition** and save it for later use.
+1. Go to [https://hub.docker.com/my-content](https://hub.docker.com/my-content).
+2. Each subscription or trial you have access to is listed. Click the **Setup**
+  button for **Docker Enterprise Edition for CentOS**.
+3. Copy the URL from  **Copy and paste this URL to download your Edition** and save it for later use.
 
-You will use this URL in a later step to create a variable called, `DOCKERURL`.
+Note that the URL will be put to use in a later step to create a variable called `DOCKERURL`.
 
 <!---
 Shared between centOS.md, rhel.md, oracle.md
 --->
 
-### Uninstall old Docker versions
+### Uninstalling All Old Docker Versions
 
-The Docker Engine - Enterprise package is called `docker-ee`. Older versions
-were called `docker` or `docker-engine`. Uninstall all older versions and
-associated dependencies. The contents of `/var/lib/docker/` are preserved,
-including images, containers, volumes, and networks. If you are upgrading from
-Docker Engine - Community to Docker Engine - Enterprise, remove the Docker
-Engine - Community package as well.
+Use the `yum remove` command to uninstall older versions and associated dependencies of Docker Engine - Enterprise (called `docker` or `docker-engine`. Note that the contents of `/var/lib/docker/` are preserved, including images, containers, volumes, and networks. In addition, if you are upgrading from Docker Engine - Community to Docker Engine - Enterprise, remove the Docker Engine - Community package.
```

Comment:
Add ")" after `docker-engine`

Reply:
You could also remove the "Note that" and start the sentence with "The contents of ..."